native/cocos/renderer/pipeline/custom/NativePipeline.cpp

```diff
@@ -1310,6 +1311,19 @@ void buildLayoutGraphNodeBuffer(
     }
 }
 
+se::Value buildRPVal;
+void buildRenderPipeline() {
+    if (buildRPVal.isUndefined()) {
```

Comment:
buildRPValue needs to be reset while `ScriptEngine` shutdowns, otherwise the after `game.restart()`, buildRPVal will keep the old value that previous `ScriptEngine` holds.
There is an interface called `addBeforeCleanupHook` that could make a hook callback which will be triggered when cleanup ScriptEngine. E.g.
```cpp
se::ScriptEngine::getInstance()->addBeforeCleanupHook([]() {
        // Doing something to release or reset your `se` resources.
});
```

Reply:
get